feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/staking/start/setupStakingType/SetupStakingTypeFragment.kt

```diff
@@ -0,0 +1,73 @@
+package io.novafoundation.nova.feature_staking_impl.presentation.staking.start.setupStakingType
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import io.novafoundation.nova.common.base.BaseFragment
+import io.novafoundation.nova.common.di.FeatureUtils
+import io.novafoundation.nova.common.mixin.impl.observeValidations
+import io.novafoundation.nova.common.utils.applyStatusBarInsets
+import io.novafoundation.nova.feature_staking_api.di.StakingFeatureApi
+import io.novafoundation.nova.feature_staking_impl.R
+import io.novafoundation.nova.feature_staking_impl.di.StakingFeatureComponent
+import io.novafoundation.nova.feature_staking_impl.presentation.staking.start.setupStakingType.adapter.EditableStakingTypeRVItem
+import io.novafoundation.nova.feature_staking_impl.presentation.staking.start.setupStakingType.adapter.SetupStakingTypeAdapter
+import kotlinx.android.synthetic.main.fragment_setup_staking_type.setupStakingTypeList
+import kotlinx.android.synthetic.main.fragment_setup_staking_type.setupStakingTypeToolbar
+
+class SetupStakingTypeFragment : BaseFragment<SetupStakingTypeViewModel>(), SetupStakingTypeAdapter.ItemAssetHandler {
+
+    companion object {
+
+        private val PAYLOAD_KEY = "payload_key"
```

Comment:
We usually name them `<Fragment name>.PAYLOAD` so this will be `SetupStakingTypeFragment .PAYLOAD`